e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -50,6 +50,26 @@ export async function setUpTestContext(
         submissionEmail: context.team.settings?.submissionEmail,
       },
     });
+
+    // has_planning_data needs to be altered manually for geospatial tests
+    await $admin.client.request(
+      gql`
+        mutation AddPlanningData($teamId: Int) {
```

Comment:
nit: I often associate "Add" with a database "insert", but this is an update - would `TogglePlanningData` be more accurate?